src/client/game/game.ts

```diff
@@ -202,18 +203,35 @@ export default class Game {
     // Register restart handler if user is an admin
     const token = bd.meetingToken;
     if (token) {
```

Comment:
The two handlers below are specifically to enable our two game host features:

* Muting all participants
* Restarting the game mid-session